app/src/main/java/com/example/android/codelabs/paging/Injection.kt

```diff
@@ -32,27 +29,19 @@ import java.util.concurrent.Executors
  */
 object Injection {
 
-    /**
-     * Creates an instance of [GithubLocalCache] based on the database DAO.
-     */
-    private fun provideCache(context: Context): GithubLocalCache {
-        val database = RepoDatabase.getInstance(context)
-        return GithubLocalCache(database.reposDao(), Executors.newSingleThreadExecutor())
-    }
-
     /**
      * Creates an instance of [GithubRepository] based on the [GithubService] and a
      * [GithubLocalCache]
      */
-    private fun provideGithubRepository(context: Context): GithubRepository {
-        return GithubRepository(GithubService.create(), provideCache(context))
+    private fun provideGithubRepository(): GithubRepository {
+        return GithubRepository(GithubService.create())
```

Comment:
I guess GithubRepository would have the same lifetime as GithubService so it doesn't really matter, but it's interesting that you have a separate provide for GithubRepository, but not for the Service